pybossa/forms/forms.py

```diff
@@ -99,9 +99,6 @@ class ProjectCommonForm(Form):
 
     password = PasswordField(
                     lazy_gettext('Password'),
-                    [pb_validator.CheckPasswordStrength(
```

Comment:
When removing this from the base class here, are there other areas that use `ProjectCommonForm` with a `PasswordField` that would no longer have a check on `CheckPasswordStrength` (since it was moved into the subclass `ProjectForm`?